src/SnackSession.js

```diff
@@ -292,6 +302,99 @@ export default class SnackSession {
     return url;
   };
 
+  /**
+   * Generates a default app.json template for the snack.
+   * @returns {Object} the app.json template
+   * @function
+   */
+  generateAppJson = (): Object => {
+    const appJsonTemplate = {
+      expo: {
+        slug: this.slug,
+        name: this.name,
```

Comment:
maybe just do the "slug" extraction here and confirm that the project belongs to someone as soon as the user starts trying to build